vendor/src/github.com/docker/libnetwork/controller.go

```diff
@@ -1003,9 +1003,9 @@ func SandboxKeyWalker(out *Sandbox, key string) SandboxWalker {
 func (c *controller) loadDriver(networkType string) error {
 	// Plugins pkg performs lazy loading of plugins that acts as remote drivers.
 	// As per the design, this Get call will result in remote driver discovery if there is a corresponding plugin available.
-	_, err := plugins.Get(networkType, driverapi.NetworkPluginEndpointType)
+	_, err := pluginstore.LookupWithCapability(networkType, driverapi.NetworkPluginEndpointType)
```

Comment:
Before doing a deep dive into the pluginstore APIs, will this call work in backward compatible way by loading lazy plugins (based of plugin-v1) ?